src/main/java/be/seeseemelk/mockbukkit/block/data/BlockDataMock.java

```diff
@@ -139,6 +139,13 @@ public boolean isFaceSturdy(@NotNull BlockFace face, @NotNull BlockSupport suppo
 
 	}
 
+	@Override
+	public boolean isRandomlyTicked()
+	{
+		// TODO Auto-generated method stub
+		throw new UnimplementedOperationException();
+	}
+
```

Comment:
Those Methods seem to be out of scope for this PR, feel free to open a separate PR to fix those